src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/DependenciesInitializer.cs

```diff
@@ -16,24 +16,36 @@ namespace Uno.UI.SourceGenerators
 {
 	internal class DependenciesInitializer
 	{
+		private static object _gate = new object();
+
 		internal static void Init()
```

Comment:
Is there a check elsewhere that this only runs once?

Reply:
It does not really need to be running once, additional runs will only return the same assemblies.